### PR TITLE
Update docs to reflect the latest 1.8 release

### PR DESCRIPTION
**Pull request by collinmesser**

### CODE REVIEW

**Comment by nathanlesage** [comment]
Thank you! Would you mind checking in with PR #78 as this appears to address some similar things? Thanks!

File: docs/en/reference/settings.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Settings
 
-Zettlr offers you a plethora of options to customise your experience with the app. It comes with three different settings dialogs that will be explained in detail here, so you know which option does what.
+Zettlr offers you a plethora of options to customise your experience with the app. It comes with three different setting dialogs that will be explained in detail here, so you know which option does what.
```
Comment (the-solipsist) [inline docs/en/reference/settings.md:3]: This seems incorrect. It should be "settings dialogs".

**Comment by the-solipsist** [comment]
Hi. @collinmesser's pul request is far more comprehensive.  However, there are a few small differences.  And then there's the stylistic choice of whether to use "File Manager mode" or "file manager mode".

**Comment by nathanlesage** [comment]
Concerning style, I'll keep myself out, you decide. In the preferences I prefer title-case (because it's more æesthetic, but in longer paragraphs of text, it might be better to use lower case, you decide!)

**Comment by collinmesser** [comment]
Sorry I haven't touched this in a bit. I'll keep working things a little bit more here and get it into a mergeable state.

**Comment by nathanlesage** [comment]
Due to the age of this PR and now existing merge conflicts, I'll close this PR now. I'd be happy to include any still relevant changes at a future point from a new PR!

**Pull request closed without merging**